jena-arq/testing/RIOT/URISchemes/file-base.rdf

```diff
@@ -0,0 +1,8 @@
+<rdf:RDF
+    xml:base="http://2dee5b0a-6100-470a-a67f-1399518cb470/nested/"
```

Comment:
Wrong URI scheme, was "file://example.com/nested/" intended?

Reply:
Commented in Java file, it was just to verify that it was not a UUID-like hostname that was causing the problem. It could be valid HTTP URL - with a local DNS search domain at least :)

Reply:
I don't see a comment about file-base.rdf being a different pattern from other "base.rdf" files. file-base.rdf is the same as http-base.rdf. 

All the other "X-base.rdf" have "X:" for xml:base.  Otherwise there is no xml:base= file: testing.

Reply:
Bah, sorry.. silly me, I thought the comment was on http-base.rdf! Fixed in 33063f335d0cb9224a64dc2f77dbaa9963d92ecb

`fileBaseRDF()` now passes after your JENA-1463 fix 1d037e80dfd8a91e9c4042f65695cadec4b17097 from #342  :-)  -- only `fileRefRDF()` still fails (passing the base to `RDFDataMgr.read`)

```turtle
<file:///example.com/nested/foo.txt> <http://www.w3.org/2000/01/rdf-schema#seeAlso> <file:///example.com/bar.txt> .
```